ggshield/cmd/secret/ignore.py

```diff
@@ -47,9 +48,10 @@ def ignore_cmd_impl(ctx: click.Context, last_found: bool) -> None:
         config = ctx.obj["config"]
         cache = ctx.obj["cache"]
         nb = ignore_last_found(config, cache)
+        path = config.config_path
+        secrets_word = pluralize("secret", nb)
```

Comment:
It's not always plural actually: it depends on `nb`. I don't think it's worth iterating too much here because the variable is only used once, one line below. Going to merge it.